terraform/projects/app-ecs-instances/instance-user-data.tpl

```diff
@@ -32,10 +32,15 @@ else
   echo "[$(date '+%H:%M:%S %d-%m-%Y')] attach-volume: /dev/$DEVICE is already formatted: $(file -s /dev/$DEVICE)"
 fi
 
-# Allow prometheus container user access to read/write/execute within container
+# Allow prometheus container user access to read/write metrics to persistent volume mount from within a container
 mkdir -p /ecs/prometheus_data
+
+#Cretaing prometheus & granting access to prometheus data files. We create the group & match it to the gid
```

Comment:
can this be simple present tense "create" and "grant" rather than "creating" and "granting"